pkg/util/log/clog.go

```diff
@@ -122,6 +122,10 @@ type loggerT struct {
 	mu struct {
 		syncutil.Mutex
 
+		// active indicates that at least one event has been logged
+		// to this logger already.
```

Comment:
Also mention the configuration changes + secondary loggers here? (basically the comment block for IsActive, which we could just point to as well.)